includes/class-stats.php

```diff
@@ -2123,7 +2123,7 @@ public function get_customer_order_count( $query = array() ) {
 					) o
 					WHERE 1=1 {$this->query_vars['status_sql']} {$user} {$customer} {$email} {$this->query_vars['where_sql']} {$this->query_vars['date_query_sql']}";
 		} else {
-			$sql = "SELECT {$function}
+			$sql = "SELECT COUNT(id)
```

Comment:
This is throwing a database error for me, but from what I can tell, it's actually due to the use of undefined variables in lines 2119 and 2122 above (`$this->query` should be `$this->query_vars`).

The error reads:
`WordPress database error You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') o
					WHERE 1=1 AND wp_edd_orders.status IN ('complete', 'revoked')' at line 6 for query SELECT COUNT(id) / total_customers AS average
					FROM wp_edd_orders
					CROSS JOIN (
						SELECT COUNT(DISTINCT customer_id) AS total_customers
						FROM 
					) o
					WHERE 1=1 AND wp_edd_orders.status IN ('complete', 'revoked')      made by require_once('wp-admin/admin.php'), do_action('download_page_edd-reports'), WP_Hook->do_action, WP_Hook->apply_filters, edd_reports_page, edd_reports_sections, EDD\Admin\Reports_Sections->display, EDD\Admin\Sections->get_all_section_contents, call_user_func_array, edd_output_report_callback, EDD\Reports\Data\Report->display, call_user_func, EDD\Reports\default_display_report, EDD\Reports\Data\Report->display_endpoint_group, call_user_func, EDD\Reports\default_display_tiles_group, EDD\Reports\Data\Tile_Endpoint->display, EDD\Reports\Data\Endpoint->display, EDD\Reports\Data\Endpoint->get_data, call_user_func, {closure}, EDD\Stats->get_customer_order_count`

and then a PHP notice (same for the two lines I mentioned):

`PHP Notice:  Undefined property: EDD\Stats::$query in \plugins\easy-digital-downloads.30\includes\class-stats.php on line 2119`

I adjust those, your updated code shows an average number of orders which makes sense to me.

Reply:
Great catch!  I'm still getting in the habit of checking logs.